sites/healthcarepackaging/server/templates/content/webinar.marko

```diff
@@ -0,0 +1,52 @@
+import hierarchyAliases from "@base-cms/marko-web/utils/hierarchy-aliases";
```

Comment:
The `hierarchyAliases` import can be removed once you remove the `$ const aliases = ` on L21